examples/standalone_panorama/example.tfvars

```
@@ -0,0 +1,56 @@
+# --- GENERAL --- #
+location              = "North Europe"
+resource_group_name   = "panorama"
+name_prefix           = "example-"
+create_resource_group = true
+tags = {
+  "CreatedBy"   = "Palo Alto Networks"
+  "CreatedWith" = "Terraform"
+}
+enable_zones = false
+
+
+
+# --- VNET PART --- #
+vnets = {
+  "vnet" = {
+    name          = "panorama-vnet"
+    address_space = ["10.1.0.0/27"]
+    network_security_groups = {
+      "management" = {
+        name = "mgmt-nsg"
+        rules = {
+          vmseries_mgmt_allow_inbound = {
+            priority                   = 100
+            direction                  = "Inbound"
+            access                     = "Allow"
+            protocol                   = "Tcp"
+            source_address_prefixes    = ["134.238.135.137", "130.41.247.15"]
+            source_port_range          = "*"
+            destination_address_prefix = "10.1.0.0/24"
+            destination_port_ranges    = ["22", "443"]
+          }
+        }
+      }
+    }
+    subnets = {
+      "management" = {
+        name                   = "mgmt-snet"
```

```suggestion
        name                   = "panorama-snet"
```
Or maybe `panorama-mgmt-snet`?

---

I was thinking about it, this is a management subnet and NSG, but in this example maybe you are right, this is the Panorama subnet and it is a dedicated Panorama example. No sense in complicating stuff.